src/client/components/pages/entity-revisions.js

```diff
@@ -78,9 +79,11 @@ class EntityRevisions extends React.Component {
 */
 	renderRevision(revision) {
 		const createdDate = new Date(revision.revision.createdAt);
-		const dateLabel = (Date.now() - createdDate < 86400000) ?
-			createdDate.toLocaleTimeString() :
-			createdDate.toLocaleDateString();
+		let includeTime = false;
+		if (Date.now() - createdDate.getTime() < 86400000) {
+			includeTime = true;
+		}
+		const dateLabel = formatDate(createdDate, includeTime);
```

Comment:
In fact it would be better to have this as a function and stick it in the helpers module since it's reused elsewhere.